src/lib.rs

```diff
@@ -30,7 +28,7 @@ extern crate std;
 pub mod crypto;
 pub mod page;
 pub mod parameters;
-pub mod quote;
+mod report;
```

Comment:
I'm not sure what's the styling guideline here, but I think this should not be in the same "group"?

Reply:
I guess styling is kinda arbitrary but I always do `mod` then `use`. I can break them up if its done otherwise